packages/arb-validator-core/message/erc721.go

```diff
@@ -32,7 +33,20 @@ type ERC721 struct {
 	Id           *big.Int
 }
 
-func (m ERC721) Equals(o ERC721) bool {
+func (m ERC721) String() string {
+	return fmt.Sprintf("ERC721(to: %v, from: %v, token: %v, id: %v)",
+		m.To,
+		m.From,
+		m.TokenAddress,
+		m.Id,
+	)
+}
+
+func (m ERC721) Equals(other Message) bool {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ERC721.Equals should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)